Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM nfcore/base
-LABEL description="Docker image containing all requirements for nf-core/atacseq pipeline"
+
+MAINTAINER Harshil Patel <harshil.patel@crick.ac.uk>
```

Comment:
MAINTAINER is deprecated, but you can add the label maintainer in LABEL

Reply:
Changed on `dev` branch through recent commit:
https://github.com/nf-core/atacseq/commit/2bde7a99a3c89b07152b6fbfbbf327cffa5acfa7

Is this ok @MaxUlysse ?